Makefile

```diff
@@ -180,3 +210,38 @@ $(EMBEDMD): $(LOCALBIN)
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+
+.PHONY: protoc
+protoc: $(PROTOC) ## download and install protoc.
+$(PROTOC): $(LOCALBIN)
+	test -s $(LOCALBIN)/protoc || \
+	curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip -o /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip
+	unzip /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip -d /tmp/protoc/ && \
+	cp /tmp/protoc/bin/protoc $(LOCALBIN)/protoc && \
+	rm -rf /tmp/protoc/
+	rm /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip
+	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION}
+
+### GCP installer targets
+
+RUN_SUBSTRATUS_INSTALLER := docker run -it \
+	-v ${HOME}/.kube:/root/.kube \
+	-e PROJECT=$(shell gcloud config get project) \
+	-e TOKEN=$(shell gcloud auth print-access-token) \
+	substratus-installer
+
+.PHONY: build_gcp_installer
+build_gcp_installer: ## Build the GCP installer.
```

Comment:
I think this should be `build_installer` ? PS: I would prefer to use `kebab-case` for Makefiles.

Reply:
Sgtm